docs/citing.rst

```diff
@@ -9,6 +9,10 @@ reproducibility of your presented results.
 pysatSpaceWeather
 -----------------
 
+The most recent citation can be found at `Zenodo
+<https://zenodo.org/record/4442679>`_.  The examples here are from the first
```

Comment:
```suggestion
<https://zenodo.org/record/3986138>`_.  The examples here are from the first
```
This should redirect to the most recent, rather than hard-wiring this to v0.0.3